apps/allocations/app/app-state-reducer.js

```diff
@@ -0,0 +1,11 @@
+function appStateReducer(state) {
+  const { github, repos } = state || {}
```

Comment:
Is this correct? It seems that github and repos should not be present on the allocations app

Reply:
Try instead `accounts` and `entries` ? (or better try to identify and confirm the needed state from previous allocations code) :)

Reply:
As far as I can see there is no need for any variables there... Testing now. I'm going to leave it in place though, as standard and one-day-might-be-needed thing?